var/spack/repos/builtin/packages/care/package.py

```diff
@@ -41,9 +41,10 @@ class Care(CMakePackage, CudaPackage, ROCmPackage):
     # package that uses cub. TODO: have all packages point to the same external
     # cub package.
     depends_on('camp+cuda', when='+cuda')
-    depends_on('umpire+cuda', when='+cuda')
+    depends_on('umpire+cuda~shared', when='+cuda')
```

Comment:
Answering a question from Kostas: I disabled shared because the umpire package has a conflict

conflicts('+shared', when='+cuda')

This could be done manually as follows:
spack install -v care@develop%clang+tests+loop_fuser+cuda cuda_arch=70 ^chai~shared ^umpire~shared ^blt@develop

But doing it here by default just seemed easier for the user.

Reply:
This constraint can be removed once we switch over fully to the new concretizer.